includes/documents/class-wcpdf-invoice.php

```diff
@@ -457,7 +457,7 @@ public function get_pdf_settings_fields( $option_name ) {
 				'args'			=> array(
 					'option_name'	=> $option_name,
 					'id'			=> 'invoice_number_search',
-					'description'   => __( 'Can potentially slow down the search process.', 'woocommerce-pdf-invoices-packing-slips' ),
+					'description'   => __( 'Can potentially slow down the search process. You can also use the HPOS(High-Performance Order Storage) feature to search orders by invoice numbers using search type selector, which is faster than this option.', 'woocommerce-pdf-invoices-packing-slips' ),
```

Comment:
Do you think it would also be good to add a link to this article? Maybe making the HPOS a hyperlink?
https://woocommerce.com/document/high-performance-order-storage/

Reply:
I agree, great suggestion.